native/tools/swig-config/gfx.i

```diff
@@ -78,6 +78,14 @@ namespace cc { namespace gfx {
 
 %ignore Device::copyBuffersToTexture;
 %ignore Device::copyTextureToBuffers;
+%ignore Device::createBuffer;
+%ignore Device::createTexture;
```

Comment:
Why are createBuffer & createTexture not needed for binding?

Reply:
> Why are createBuffer & createTexture not needed for binding?

Just sync`skip Device::[Device copyBuffersToTexture copyTextureToBuffers createBuffer createTexture getInstance setOptions getOptions]` in gfx.ini